src/Models/Bohrprofil.cs

```diff
@@ -86,45 +86,25 @@ public class Bohrprofil : EwsModelBase
         /// Foreign Key: ID des Codetyps für Feld quali.
         /// </summary>
         [Column("h_quali")]
-        public int HQualitaetId { get; set; }
-
-        /// <summary>
-        /// Codetyps für Feld quali.
-        /// </summary>
-        public Code HQualitaet { get; set; }
+        public int HQualitaet { get; set; }
 
         /// <summary>
         /// Foreign Key: ID des Codetyps für Feld tektonik.
         /// </summary>
         [Column("h_tektonik")]
-        public int HTektonikId { get; set; }
-
-        /// <summary>
-        /// Codetyps für Feld tektonik.
-        /// </summary>
-        public CodeTyp HTektonik { get; set; }
+        public int HTektonik { get; set; }
 
         /// <summary>
         /// Foreign Key: ID des Codetyps für Feld fmfelso.
         /// </summary>
         [Column("h_fmfelso")]
-        public int HFormationFelsId { get; set; }
-
-        /// <summary>
-        /// Codetyp für Feld fmfelso.
-        /// </summary>
-        public CodeTyp HFormationFels { get; set; }
+        public int HFormationFels { get; set; }
 
         /// <summary>
         /// Foreign Key: ID des Codetyps für Feld fmeto.
         /// </summary>
         [Column("h_fmeto")]
-        public int HFormationEndtiefeId { get; set; }
-
-        /// <summary>
-        /// Codetyp für Feld fmeto.
-        /// </summary>
```

Comment:
Q: Ist dieser Change gewollt?

Reply:
Ja das braucht es nicht mehr. Bei der Bohrung hatte ich es schon umgestellt. Aber es gehört eigentlich nicht in diesen change. Ich mache es dann mit der Bohrprofilmaske.